compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/symbols/BallerinaStreamTypeSymbol.java

```diff
@@ -50,24 +52,24 @@ public TypeSymbol typeParameter() {
     }
 
     @Override
-    public TypeSymbol completionValueTypeParameter() {
+    public Optional<TypeSymbol> completionValueTypeParameter() {
         if (this.completionValueTypeParameter == null) {
             BType completionType = ((BStreamType) this.getBType()).error;
-            TypesFactory typesFactory = TypesFactory.getInstance(this.context);
-            this.completionValueTypeParameter = typesFactory.getTypeDescriptor(completionType);
+            if (completionType != null) {
```

Comment:
This won't be null right? So we don't have to make completion type an Optional<TypeSymbol> right?